Makefile

```diff
@@ -135,3 +143,31 @@ upload_package:
 clear_package_data:
 	rm -rf build dist/* ondewo_nlu_client.egg-info
 
+ondewo_release: spc clone_devops_accounts run_release_with_devops ## Release with credentials from devops-accounts repo
+	@rm -rf ${DEVOPS_ACCOUNT_GIT}
+
+clone_devops_accounts: ## Clones devops-accounts repo
+	if [ -d $(DEVOPS_ACCOUNT_GIT) ]; then rm -Rf $(DEVOPS_ACCOUNT_GIT); fi
+	git clone git@bitbucket.org:ondewo/${DEVOPS_ACCOUNT_GIT}.git
+
+DEVOPS_ACCOUNT_GIT="ondewo-devops-accounts"
+DEVOPS_ACCOUNT_DIR="./${DEVOPS_ACCOUNT_GIT}/info"
+
+
+TEST:
+	@echo ${GITHUB_GH_TOKEN}
+	@echo ${PYPI_USERNAME}
+	@echo ${PYPI_PASSWORD}
+
+
+run_release_with_devops:
+	$(eval info:= $(shell cat ${DEVOPS_ACCOUNT_DIR} | grep GITHUB & cat ${DEVOPS_ACCOUNT_DIR} | grep PYPI_USERNAME & cat ${DEVOPS_ACCOUNT_DIR} | grep PYPI_PASSWORD))
+	make release $(info)
+
+spc: ## Checks if the Release Branch, Tag and Pypi version already exist
```

Comment:
Didnt know what to name the make target, so i took the short version of "some preventive checks"